lib/src/talsec.dart

```diff
@@ -106,6 +108,26 @@ class Talsec {
     );
   }
 
+  /// Adds [packageName] to the whitelist.
+  ///
+  /// Once added, the package will be excluded from the list of blocklisted
+  /// packages and won't appear in the list of suspicious applications in
+  /// the future detections.
+  ///
+  /// **Adding package is one-way process** - to remove the package from the
+  /// whitelist, you need to remove application data or reinstall the
+  /// application.
+  Future<void> addToWhitelist(String packageName) async {
+    if (!Platform.isAndroid) {
+      return;
```

Comment:
The question is whether "graceful" return is a good option. Let's sync about this :D